langstream-webservice/src/main/java/ai/langstream/webservice/common/TenantResource.java

```diff
@@ -62,6 +69,42 @@ TenantConfiguration getTenant(@NotBlank @PathVariable("tenant") String tenant) {
         return configuration;
     }
 
+    @PostMapping(value = "/{tenant}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a tenant")
+    void createTenant(
+            @NotBlank @PathVariable("tenant") String tenant,
+            @RequestBody CreateTenantRequest createTenantRequest)
+            throws TenantException {
+        try {
+            globalMetadataService.createTenant(tenant, createTenantRequest);
```

Comment:
we need an hard limit for the "--max-total-resource-units", configurable in the control plane